opennms-base-assembly/src/main/filtered/bin/send-event.pl

```diff
@@ -56,7 +56,7 @@
 if ($help)     { print get_help(); exit; }
 
 # parm array is numerically referenced in OpenNMS' templates
-@PARMS = reverse map { parse_parm($_) } @PARMS;
+@PARMS = map { parse_parm($_) } @PARMS;
```

Comment:
@j-white I am not sure why this code reverts the order of the parameters that we send. It will mean that we will save them in reverse order. I think it should be
```@PARMS = map { parse_parm($_) } @PARMS;``` instead of ```@PARMS = reverse map { parse_parm($_) } @PARMS;```

Reply:
Not sure either,  looks like it's been that way for > 12 years though.

Keeping the order specified by the user makes sense to me.

Reply:
ok, then let's leave it with my change.